manim/camera/camera.py

```diff
@@ -1208,30 +1207,33 @@ def resize_background_array_to_match(self, background_array, pixel_array):
         mode = "RGBA" if pixel_array.shape[2] == 4 else "RGB"
         return self.resize_background_array(background_array, width, height, mode)
 
-    def get_background_array(self, file_name):
+    def get_background_array(self, image: Union[Image.Image, str]):
```

Comment:
what about `typing.Union[Image.Image,str, pathlib.Path]` ?
But I did not test if pathlib works here.

Reply:
They should works since they are immutable and hashable.

Reply:
Perfect.
I will try to review and test it later today, but maybe also only next week.